pb/ctrl_pb/ctrl.proto

```diff
@@ -39,6 +40,7 @@ message BindRequest {
     BindType bindType = 1;
     string token = 2;
     string serviceId = 3;
+    bytes pubkey = 4;
```

Comment:
should this be more generic like SessionRequest? or vice-a-versa?

Reply:
Paul and I were just discussing that we need to revisit the generalized "bind" concept in the fabric at some point in the next chunk of time (#38).

I think this is probably okay as `bytes` for now.

Reply:
yep, I suspected as much, especially when we start thinking about clustered hosting